src/plugins/pcr_primer_design_for_dna_assembly/src/PCRPrimerDesignForDNAAssemblyOPWidget.cpp

```diff
@@ -274,7 +274,9 @@ void PCRPrimerDesignForDNAAssemblyOPWidget::sl_selectManually() {
     auto sequenceSelection = annDnaView->getActiveSequenceContext()->getSequenceSelection();
     int startRegionValue = sbStartRegion->value();
     int endRegionValue = sbEndRegion->value();
-    sequenceSelection->setSelectedRegions({ U2Region(startRegionValue, endRegionValue - startRegionValue) });
+    if (startRegionValue > 0) {
```

Comment:
In what case `startRegionValue` less than 0?

Reply:
With the current changes, the invalid region is not displayed in the Sequence View, before the changes, the region went beyond sequence boundary.